### DISCUSSION TITLE
Issue with Secure connection over SMTP

### DISCUSSION THREAD

**Discussion by Nillduke** [postalserver/postal#2156] [Development discussions]
I have installed SSL (let's encrypt)

![image](https://user-images.githubusercontent.com/9651337/189957109-af570280-63dc-4cff-bad1-57d13a301254.png)

But my mails are not going under secure conection

![image](https://user-images.githubusercontent.com/9651337/189954522-1e604372-0bb2-4e5a-83b4-8090b69cbb93.png)

No error on logs file..

**Comment by kluvi** [comment]
I am not sure, how do you test it (SSL/TLS or STARTTLS?) but maybe this helps you: https://github.com/postalserver/postal/discussions/1757#discussioncomment-3523525

**Comment by willpower232** [comment]
You have to configure secure SMTP in the config file as you need to give Postal a certificate file to enable it.
